src/main/java/eu/usrv/enhancedlootbags/core/items/ItemLootBag.java

```diff
@@ -89,7 +88,8 @@ public String getItemStackDisplayName(ItemStack pStack) {
         if (pStack.getItemDamage() == 0) tInnerName = StatHelper.get("string.default");
         else {
             LootGroup tGrp = _mLGHandler.getGroupByIDClient(pStack.getItemDamage());
-            tInnerName = tGrp == null ? "Error" : tGrp.getGroupName();
+            String tLocale = StatHelper.get("group." + tGrp.getGroupName().toLowerCase().replaceAll("\\s+", "_"));
```

Comment:
Please don't use `StatHelper` directly for translation group names since it requires lang entries to be in the "enhancedlootbags" namespace. You should instead directly use `StatCollector::translateToLocal` on the name which can be made to include the right namespace.

Since the lootbag entry will need to be the full lang key, let's also drop the "group" infix. This has the side benefit that it's easier to see where a certain lang key is defined and used when you always use it as a single unbroken string instead of concatenating it.

Reply:
so in my PR to the main repo we directly use the locale key for `GroupName`s in the xml right?

Reply:
Yup, we already have the MetaID as a unique identifier so the name being a full lang key is very suitable.